tests/functional/issues-non-auth.js

```diff
@@ -101,6 +101,15 @@ define([
           assert.notInclude(className, "wc-Comment-content-nsfw--display");
         })
         .end();
+    },
+
+    "Issue page title matches actual issue number": function() {
```

Comment:
Could you test with an array of (routes, expected_title) so we are sure to cover all the modified pages.